webroot/src/store/user/user.spec.ts

```diff
@@ -159,7 +168,7 @@ describe('User Store Actions', async () => {
 
         expect(commit.calledOnce).to.equal(true);
         expect(commit.firstCall.args).to.matchPattern([MutationTypes.LOGIN_SUCCESS]);
-        expect(dispatch.calledOnce).to.equal(true);
+        expect(dispatch.calledOnce).to.equal(false);
```

Comment:
What changed here?

Reply:
The dispatched account fetch is no longer called (for the time being). That was the commented out line that we just removed.